integration-tests/test/basic-l1-l2-communication.spec.ts

```diff
@@ -56,23 +59,35 @@ describe('Basic L1<>L2 Communication', async () => {
         const value = `0x${'77'.repeat(32)}`
 
         // Send L2 -> L1 message.
-        const transaction = await env.l2Messenger.sendMessage(
-          L1SimpleStorage.address,
-          L1SimpleStorage.interface.encodeFunctionData('setValue', [value]),
-          5000000,
+        const transaction = await env.messenger.sendMessage(
```

Comment:
Note to self: maybe worth having `sendMessageToL1` and `sendMessageToL2` functions to avoid needing the `direction` parameter